### ISSUE TITLE
War Attack Reminder 

### ISSUE THREAD

**Issue by iAmCLY** [raysun/SidekickIssues#35]
<img width="1167" alt="screen shot 2017-08-24 at 1 55 10 pm" src="https://user-images.githubusercontent.com/26148953/29688233-e233de5c-88d3-11e7-9b60-0964ff41ca2f.png">

**Comment by raysun** [comment]
Yep I've seen this rule in a few clans now so it's a good candidate and easy

**Comment by unseenmagik** [comment]
This would be a great benefit to our clan also. TH8/9 attack before 10 and 10 before 11 etc.

**Comment by IamCLY** [comment]
Added

**Comment by renaatski** [comment]
just had the same idea request, so I'm tagging it on to here as well.. It would truly be a great addition to the bot, and relieve a lot of the burden of clan leadership for chasing ppl to do their attacks, etc...

Our clan uses something along the lines of:

* TH9 are expected to attacking within the first 12 hours of war
* TH10 are expected to attack within the first 18 hours of war, ideally waiting up on the TH9, but not obligatory
* a final call when war reaches its last hour maybe?

so when sidekick would notice someone hasn't attacked yet and the (admin-configured) threshold has been reached, notify the ppl that have to perform their attacks still (and not @ tagging the ones that have!)

Ideally, the admin should be able to configure these thresholds on a per clan/channel basis, or update them when needed

/attack_reminder \<th-level> \<time-threshold>

where time-threshold could be in minutes (for shorter wars, but those require more attention of the particpants anyway already)  or hours    e.g. [1-23, off]

* e.g. configure th9 attack reminders to occur after 12h into war
/attack_reminder_th9 12
* e.g. configure th10 attack reminders to occur after 18h into war
/attack_reminder th10 18
* e.g. configure a final call reminders to all players not having attacked when war reaches its final hour
/attack_reminder final_call 1

* e.g. no attack reminders for TH9
/attack_reminder th9 off


optionally you could already do the stuff for th11 too, as th12 may exist at some point!

etc..
